TRANSLATING.md

```diff
@@ -0,0 +1,142 @@
+
+### Client part
+
+Creating a translation for the client part is very similar to the translation for the server part. 
+
+Create a copy of the file `english.ts` from and to the folder `src/client/app/strings` and rename this copy to your language, like you did for the server part. Open the new file and rename the `getEnglishStrings()` to the same function name, you used in the server part. 
+
+Then open the file `index.ts` in the client part folder `src/client/app/strings` and - in this case - add three lines of code. If we keep our example of adding a french translation, the code will look similar to the following code block:
+
+```ts
+import { getEnglishStrings } from './english';
+import { getGermanStrings } from './german';
+import { getFrenchStrings } from './french';
+
+let availableLanguages = [
+    { name: 'English', value: 'en' },
+    { name: 'Deutsch', value: 'de' },
+    { name: 'Français', value: 'fr' }
+];
+
+const languageGenerator: { [key: string]: () => ClientStrings } = {
+    'en': () => getEnglishStrings(),
+    'de': () => getGermanStrings(),
+    'fr': () => getFrenchStrings(),
+    '??': () => createFakeStrings(() => getEnglishStrings())
+};
+```
+
+Like for the server part, the two lines `import { getFrenchStrings } from './french';` and `'fr': () => getFrenchStrings(),` have been added to the code. Additionally add the new line  `{ name: 'Français', value: 'fr' }` after the last entry for the list of available languages. Please do not forget to add the comma at the end of previous line.
+
+### Manifests
+
+The translations of strings within the manifest files are addressed by the XML tag `override` and an indication of the country code. So, for translating just duplicate a line from another translation, set your country code and update the text.
+
+Most translatable strings are located at the bottom of the XML files. An exception is the description of Script Lab, located on top of the XML file.
+
+Here is an example, where the string *Help* has been translated to German and French.
+
+```xml
+  <bt:String id="PG.RunCommand.Title" DefaultValue="Help">
+    <Override Locale="de" Value="Hilfe"/>
+    <Override Locale="fr" Value="Aide"/>
+  </bt:String>
+```
+
+<a id="improve"></a>
+## Improve an existing translation
+
+If you would like to improve an existing translation, e.g. correct a typing error or suggest a better wording, then just open the corresponding existing language files and do your changes. Please note, that when modifying one of the manifest files, you should also do the change in the 3 other files.
+
+For example, if you discovered an error in the translation to *German* for the *client part*, then open the file `german.ts` from the folder `src/client/app/strings` and do the change.
```

Comment:
I would move that sentence up by one, and the manifest sentence down by own to a separate paragraph.  Otherwise I mistakenly thought that this sentence is also about the manifest.